fix-AppData-file.patch

```diff
@@ -2,14 +2,11 @@ diff --git a/data/supertuxkart.appdata.xml b/data/supertuxkart.appdata.xml
 index 11240dc45..2284e3a1c 100644
 --- a/data/supertuxkart.appdata.xml
 +++ b/data/supertuxkart.appdata.xml
-@@ -50,4 +50,10 @@
-   <url type="donation">https://supertuxkart.net/Donate</url>
-   <url type="help">https://supertuxkart.net/Community</url>
-   <url type="translate">https://supertuxkart.net/Translating_STK</url>
-+  <content_rating type="oars-1.1">
-+    <content_attribute id="violence-cartoon">mild</content_attribute>
-+  </content_rating>
+@@ -54,4 +54,7 @@
+     <content_attribute id="violence-cartoon">mild</content_attribute>
+     <content_attribute id="social-chat">intense</content_attribute>
+   </content_rating>
 +  <releases>
-+    <release version="1.0" date="2019-04-21"/>
++    <release version="1.1" date="2020-01-05"/>
```

Comment:
Please, also keep the old version for history. :-)

```
<release version="1.1" date="2020-01-05"/>
<release version="1.0" date="2019-04-21"/>
```

Otherwise, LGTM! Thanks!